build.gradle.kts

```diff
@@ -22,8 +22,8 @@ repositories {
 
 dependencies {
     implementation("xpp3:xpp3:1.1.4c")
-    implementation("org.smali:smali:2.5.2")
-    implementation("app.revanced:multidexlib2:2.5.3-8649b642")
+    implementation("app.revanced:smali:2.5.3-a3836654")
+    implementation("app.revanced:multidexlib2:2.5.3-a3836654")
     implementation("org.apktool:apktool-lib:2.9.0-SNAPSHOT")
```

Comment:
you forgot to change this to change from org.apktool to app.revanced.

Reply:
should we add something like this https://github.com/revanced/revanced-patcher/blob/f6d1fd144ef8cb876df8a638589ae4e1eb28c643/build.gradle.kts#L14-L20 for smali and apktool-lib too?

Reply:
It automatically authenticates with maven, theres no need to add multiple for them.